mappings/net/minecraft/network/MessageType.mapping

```diff
@@ -1,82 +1,148 @@
-			COMMENT previous ones.
+		METHOD method_43850 (Lnet/minecraft/class_2561;Lnet/minecraft/class_7436;Lnet/minecraft/class_7463;)Lnet/minecraft/class_2561;
+			ARG 2 decoration
+		CLASS class_7467 Priority
+			COMMENT The priority of narration. When narrating messages that have priority, it will
+			COMMENT interrupt the current narrations. This is also used to determine if something
+			COMMENT {@linkplain net.minecraft.client.option.NarratorMode#shouldNarrate should be
+			COMMENT narrated} when the narrator option is set to "Chat" or "System".
 			FIELD field_39236 CODEC Lcom/mojang/serialization/Codec;
-				COMMENT The codec for the narration kind.
 			FIELD field_39237 name Ljava/lang/String;
-			FIELD field_39238 interrupt Z
+			FIELD field_39238 prioritized Z
 			METHOD <init> (Ljava/lang/String;ILjava/lang/String;Z)V
 				ARG 3 name
-				ARG 4 interrupt
-			METHOD method_43851 shouldInterrupt ()Z
+				ARG 4 prioritized
+			METHOD method_43851 hasPriority ()Z
+				COMMENT {@return whether the message has priority over others and should interrupt
+				COMMENT their narrations}
 	CLASS class_7468 DisplayRule
-		COMMENT A display rule for a message.
+		COMMENT The display rule for the message type. This contains the decoration applied
+		COMMENT to the message.
 		FIELD comp_797 Ljava/util/Optional;
-			COMMENT the formatting of a message, or {@code Optional.empty()} if the content
-			COMMENT is displayed directly
+			COMMENT the decoration applied to a message, or {@link Optional#empty()} if the content
+			COMMENT is displayed as is
 		FIELD field_39240 CODEC Lcom/mojang/serialization/Codec;
-			COMMENT The codec for the message display rule.
 		METHOD method_43853 of ()Lnet/minecraft/class_2556$class_7468;
-			COMMENT {@return a rule that displays a message's content directly} The message
-			COMMENT is displayed without any sender-related information.
+			COMMENT {@return the display rule with no decoration}
+			COMMENT
+			COMMENT <p>This does not mean the message is not displayed; this means that the message
+			COMMENT is displayed as is without any extra processing.
 		METHOD method_43854 (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 			ARG 0 instance
 		METHOD method_43855 of (Lnet/minecraft/class_7463;)Lnet/minecraft/class_2556$class_7468;
-			COMMENT {@return a rule to format the message} It allows displaying the sender
-			COMMENT information in addition to the message content.
-			ARG 0 format
-		METHOD method_43856 apply (Lnet/minecraft/class_2561;Lnet/minecraft/class_7436;)Lnet/minecraft/class_2561;
-			COMMENT {@return the text to display after applying this rule on the given
-			COMMENT content and sender}
-			ARG 1 content
+			COMMENT {@return the display rule with the specified decoration}
+			ARG 0 decoration
+		METHOD method_43856 applyDecoration (Lnet/minecraft/class_2561;Lnet/minecraft/class_7436;)Lnet/minecraft/class_2561;
```

Comment:
I suggest just calling it `apply` no matter you name this `decoration` or `display rule`